.gitattributes

```diff
@@ -1,18 +1,8 @@
 docker/**/*.sh text eol=lf
 *.mp4 filter=lfs diff=lfs merge=lfs -text
-docs/static/img/hero-screenshot.jpg filter=lfs diff=lfs merge=lfs -text
-docs/static/img/sql_lab.jpg filter=lfs diff=lfs merge=lfs -text
-docs/static/img/dashboard.jpg filter=lfs diff=lfs merge=lfs -text
-docs/static/img/dashboard_card_view.jpg filter=lfs diff=lfs merge=lfs -text
-docs/static/img/data-point.jpg filter=lfs diff=lfs merge=lfs -text
-docs/static/img/explore.jpg filter=lfs diff=lfs merge=lfs -text
-docs/static/img/explore_ui.jpg filter=lfs diff=lfs merge=lfs -text
-docs/static/img/grid-background.jpg filter=lfs diff=lfs merge=lfs -text
-docs/static/img/github.png filter=lfs diff=lfs merge=lfs -text
-docs/static/img/roadmap.png filter=lfs diff=lfs merge=lfs -text
-docs/static/img/root-cert-example.png filter=lfs diff=lfs merge=lfs -text
-docs/static/img/applitools.png filter=lfs diff=lfs merge=lfs -text
-docs/static/img/github-dark.png filter=lfs diff=lfs merge=lfs -text
 RELEASING/**/*.jpg filter=lfs diff=lfs merge=lfs -text
 RELEASING/**/*.png filter=lfs diff=lfs merge=lfs -text
 RELEASING/**/*.gif filter=lfs diff=lfs merge=lfs -text
+docs/static/img/**/*.jpg filter=lfs diff=lfs merge=lfs -text
```

Comment:
Hah, I was just going to ask this supports wildcards. Not sure we even need the file extensions here, as people might add svg, webp, mov, mp4, who knows.

Reply:
Right, I could be less specific here and include the `docs/static/img/*`. For RELEASING I could probably find a better glob too, but it feels like this works well enough. Goal is to get the bulk of the binaries out. 

I terms of doing it for the whole repo, it can create confusion for people who don't have git-lfs installed, and it can become hard to define which process in CI need or don't need which part of the lfs. For now I was able to switch off lfs downloads for everything EXCEPT docs building and docker building (because of the geojson, we do want that when building dockers).